content/en/docs/contribute/style/style-guide.md

```diff
@@ -376,7 +374,7 @@ Remember that screen readers are linear output devices which means they output i
   <tr><th>Do</th><th>Don't</th></tr>
   <tr><td>Use only one first-level heading per page.</td><td>Use first level heading in blog posts, as Hugo will automatically convert the title section of your post’s front matter into a first-level heading.</td></tr><tr><td>Use ordered headings to provide a meaningful high-level outline of your content.</td><td>Use headings level 4 through 6, unless it is absolutely necessary. If your content is that detailed, it may need to be broken into separate articles.</td>
   <tr><td>Use pound or hash signs (#) for non-blog post content.</td><td> Use underlines (--- or ===) to designate first-level headings.</td></tr> 
-  <tr><td>Use sentence case for headings. For example, **Extend kubectl with plugins**</td><td>Use title case for headings. For example, **Extend Kubectl With Plugins** </td></tr>  
+  <tr><td>Use sentence case for headings. For example, <b>Extend kubectl with plugins</b></td><td>Use title case for headings. For example, <b>Extend Kubectl With Plugins</b></td></tr>  
```

Comment:
Hello @Rajakavitha1. I read through the content again, in detail, to make sure that I could follow the guidelines. Here are a few areas that may need clarification.

- Line breaks, line 367. I don't fully understand the second sentence about `The exception is second-level headings, where it should be two newlines`
- Headings, line 371, reword or break up the sentences:
Remember that screen readers are linear output devices, which means that they output items on a page one at a time.
If there is a lot of content on your page, applying headings gives the page a structure that allows the user to easily navigate the page[, filtering on topics of interest].
- Headings, line 375, where do we explicitly use `<h1>` or `#`? The hugo template converts the title to the h1 header for all doc pages?
- Paragraphs, line 380, Try to keep paragraphs under 6 sentences.
- Links, line 388, links could show how to code a link, and then show the output. I think this technique is useful. See the example of `Ordered lists` on this page. The example is set up like a cookbook.
- Lists, line 394, `The user can then use the arrow keys to move up and down`. I am not sure about `to cursor`.
- Lists, note, ```{{< note >}} If a vertical list completes a sentence begun in the introduction to the list, the final period is omitted unless the items in the list are separated by commas or semicolons.{{< /note >}}```

This note seems confusing. I think you are saying that it is okay to start a sentence and then add list items, as part of the sentence, where each item ends with a comma or semicolon.

- Tables, line 414, this could be shortened, as the table caption link explains the shortcode.
The semantic purpose of an [html] table is to present tabular data. Sighted users can quickly scan the table but a screen reader goes through line by line[ this is repeated from above]. 

->>What does the table caption do to help the screen reader interpret the output?

[Proper markup must be added to help the screen reader make the correct associations that a sighted user would.]
- Add table caption shortcode. See, [Table captions](/docs/contribute/style/hugo-shortcodes/#table-captions).

Reply:
Thanks @kbhawkey !!!! As usual, I appreciate the time and efforts you take in reviewing the PR. YOur comments are invaluable. I will look into each of these and commit the changes soon. However, if there is any point that I needs a discussion I will catch up with you on slack.

Reply:
> * Line breaks, line 367. I don't fully understand the second sentence about `The exception is second-level headings, where it should be two newlines`

H2 usually follows the H1 (or the title) without any preceding paragraphs or texts. A two line spacing helps visualize the overall structure of content in a code editor better.

Reply:
> his note seems confusing. I think you are saying that it is okay to start a sentence and then add list items, as part of the sentence, where each item ends with a comma or semicolon.

I meant: Ordered lists that are part of an incomplete introductory sentence can be in lowercase and punctuated as if each item was a part of the introductory sentence.

Reply:
> ->>What does the table caption do to help the screen reader interpret the output?

Imagine reading an entire table cell by cell and then piecing together what the table is trying to tell you.